docker/standalone/docker-compose.yml

```diff
@@ -8,15 +8,16 @@ networks:
       name: ortelius_services
 services:
   avalanche:
+    env_file:
+      - standalone.env
     image: "avaplatform/avalanchego:v1.0.5"
-    command: [
-      "./build/avalanchego",
-      "--network-id=5",
-      "--db-dir=/var/lib/avalanche",
-      "--log-level=info",
-      "--http-host=0.0.0.0",
-      "--ipcs-chain-ids=11111111111111111111111111111111LpoYY,2vrXWHgGxh5n3YsLHMV16YVVJTpT4z45Fmb4y3bL6si8kLCyg9",
-    ]
+    command: bash -cx "env && ./build/avalanchego
```

Comment:
Is this bash env command really needed for this to work? The envs from the file should be already populated right?

Reply:
added env for debugging..